src/ttrssapi.cpp

```diff
@@ -416,15 +416,16 @@ rsspp::Feed TtRssApi::fetch_feed(const std::string& id, CURL* cached_handle)
 
 			if (!item_obj["attachments"].is_null()) {
 				if (item_obj["attachments"].size() >= 1) {
-					json a =
-						item_obj["attachments"].front();
-					if (!a["content_url"].is_null()) {
-						item.enclosure_url =
-							a["content_url"];
-					}
-					if (!a["content_type"].is_null()) {
-						item.enclosure_type =
-							a["content_type"];
+					for (auto it = item_obj["attachments"].begin();
```

Comment:
I think this can be simplified:
1. it's no longer necessary to run the `size() >= 1` check — if the "attachments" array is empty, the `for` loop will simply do nothing;
2. the `it` iterator is not manipulated inside the loop, so a range-based for loop can be used instead:
    ```c++
    for (const json& a : item_obj["attachments"]) {
    ```

Reply:
Thanks for pointing this out， I will simplify this